recipes/forcebalance/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ForceBalance" %}
+{% set version = "1.7.3" %}
+{% set sha256 = "d662e48862c861af6b0bc1c917ef90c65a51ad7102368b0c3945c9aff3450621" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/leeping/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - {{ pin_compatible('numpy', lower_bound='1.14') }}
+    - scipy
```

Comment:
Do you link against SciPy?

Reply:
No, but I need it available before building because there's a `try: import` check in setup.py (until the next release)

Reply:
Ahh ok!